code/game/objects/effects/credits.dm

```diff
@@ -0,0 +1,46 @@
+#define CREDIT_ROLL_SPEED 150
+#define CREDIT_SPAWN_SPEED 30
+#define CREDIT_ANIMATE_HEIGHT (14 * TURF_PIXEL_DIAMETER)
+#define CREDIT_EASE_DURATION 30
+#define CREDITS_LOC locate(12, 167, ZLEVEL_CENTCOM)
+
+/proc/RollCredits()
+    var/icon/credits_file = new('icons/credits.dmi')
+    var/list/contributors = icon_states(credits_file) - null
+    qdel(credits_file)

```

Comment:
This could have just been var/list/contributors = icon_states('icons/credits.dmi') - null